2024-12-04 13:00:07,722 - app.app_main - DEBUG - Logging has been configured with basicConfig
2024-12-04 13:00:08,077 - app.app_main - INFO - Application is starting up
2024-12-04 13:00:08,952 - app.routes - DEBUG - Root Path: /peanut
2024-12-04 13:00:24,849 - app.routes - INFO - Received request to run pipeline
2024-12-04 13:00:24,853 - app.routes - INFO - Files: ['GSE5281EC_KEGG_ALZHEIMERS_DISEASE.xlsx'], Test Name: multiple, Species: H_sapiens, Alpha: 1.0, Network: Anat, Pathway File: kegg, Min Genes per Pathway: 15, Max Genes per Pathway: 500, FDR Threshold: 0.05
2024-12-04 13:00:24,870 - app.utils - INFO - Validating uploaded file
2024-12-04 13:00:38,816 - app.utils - INFO - File preprocessed successfully
2024-12-04 13:00:38,817 - app.utils - INFO - File validation successful
2024-12-04 13:00:46,005 - pipeline.pipeline_main - ERROR - An error occurred in pipeline_main: Invalid file path or buffer object type: <class 'tuple'>
2024-12-04 13:00:46,005 - app.pipeline_runner - ERROR - Error in pipeline execution for job_code f87f72f8: Invalid file path or buffer object type: <class 'tuple'>
Traceback (most recent call last):
  File "/home/yair/projects/Yair_propagation/app/pipeline_runner.py", line 28, in run_pipeline
    output_file_path = await pipeline_main(conditions_data, settings)
  File "/home/yair/projects/Yair_propagation/pipeline/pipeline_main.py", line 63, in pipeline_main
    condition_data_df = pd.read_excel(condition_path)
  File "/home/yair/.local/lib/python3.10/site-packages/pandas/io/excel/_base.py", line 495, in read_excel
    io = ExcelFile(
  File "/home/yair/.local/lib/python3.10/site-packages/pandas/io/excel/_base.py", line 1550, in __init__
    ext = inspect_excel_format(
  File "/home/yair/.local/lib/python3.10/site-packages/pandas/io/excel/_base.py", line 1402, in inspect_excel_format
    with get_handle(
  File "/home/yair/.local/lib/python3.10/site-packages/pandas/io/common.py", line 728, in get_handle
    ioargs = _get_filepath_or_buffer(
  File "/home/yair/.local/lib/python3.10/site-packages/pandas/io/common.py", line 472, in _get_filepath_or_buffer
    raise ValueError(msg)
ValueError: Invalid file path or buffer object type: <class 'tuple'>
2024-12-04 13:00:46,012 - app.routes - DEBUG - Checking status for job_code: f87f72f8
2024-12-04 13:00:46,012 - app.routes - DEBUG - Job data: {'status': 'Failed', 'output_file': None, 'expiry': datetime.datetime(2024, 12, 11, 13, 0, 24, 854801), 'error': "Invalid file path or buffer object type: <class 'tuple'>"}
2024-12-04 13:02:14,144 - app.app_main - INFO - Application is shutting down
2024-12-04 13:02:21,722 - app.app_main - DEBUG - Logging has been configured with basicConfig
2024-12-04 13:02:21,735 - app.app_main - INFO - Application is starting up
2024-12-04 13:02:23,644 - app.app_main - INFO - Application is shutting down
2024-12-04 13:02:27,833 - app.app_main - DEBUG - Logging has been configured with basicConfig
2024-12-04 13:02:27,859 - app.app_main - INFO - Application is starting up
2024-12-04 13:02:34,883 - app.routes - DEBUG - Root Path: /peanut
2024-12-04 13:02:44,919 - app.routes - INFO - Received request to run pipeline
2024-12-04 13:02:44,920 - app.routes - INFO - Files: ['GSE5281HIP_KEGG_ALZHEIMERS_DISEASE.xlsx'], Test Name: multiple, Species: H_sapiens, Alpha: 1.0, Network: Anat, Pathway File: kegg, Min Genes per Pathway: 15, Max Genes per Pathway: 500, FDR Threshold: 0.05
2024-12-04 13:02:44,922 - app.utils - INFO - Validating uploaded file
2024-12-04 13:02:47,577 - app.utils - INFO - File preprocessed successfully
2024-12-04 13:02:47,577 - app.utils - INFO - File validation successful
2024-12-04 13:02:52,438 - pipeline.propagation_routines - INFO - Running GSEA without propagation
2024-12-04 13:02:52,440 - pipeline.propagation_routines - INFO - Processing all genes without network restriction.
2024-12-04 13:03:23,880 - pipeline.pathway_enrichment - INFO - Completed Mann-Whitney U test and ranking for 389 pathways.
2024-12-04 13:03:24,717 - pipeline.pathway_enrichment - INFO - Enrichment analysis completed. Results saved to /home/yair/projects/Yair_propagation/pipeline/Outputs/multiple_GSE5281HIP_KEGG_ALZHEIMERS_DISEASE_04_12_2024__13_02_52.xlsx.
2024-12-04 13:03:30,922 - pipeline.pipeline_main - INFO - Results archived at: /home/yair/projects/Yair_propagation/pipeline/Outputs/Archives/multiple_results.zip
2024-12-04 13:03:31,706 - app.pipeline_runner - DEBUG - Pipeline main returned output_file_path: /home/yair/projects/Yair_propagation/pipeline/Outputs/Archives/multiple_results.zip
2024-12-04 13:03:31,707 - app.pipeline_runner - DEBUG - Relative output path: pipeline/Outputs/Archives/multiple_results.zip
2024-12-04 13:03:31,708 - app.pipeline_runner - INFO - Job 0ddf208d finished successfully.
2024-12-04 13:03:31,723 - app.routes - DEBUG - Checking status for job_code: 0ddf208d
2024-12-04 13:03:31,723 - app.routes - DEBUG - Job data: {'status': 'Finished', 'output_file': 'pipeline/Outputs/Archives/multiple_results.zip', 'expiry': datetime.datetime(2024, 12, 11, 13, 2, 44, 920748)}
2024-12-04 13:03:31,724 - app.routes - DEBUG - Download URL for job 0ddf208d: /peanut/download-file/pipeline/Outputs/Archives/multiple_results.zip
2024-12-04 13:03:34,263 - app.routes - INFO - User requested file: pipeline/Outputs/Archives/multiple_results.zip
2024-12-04 13:03:34,264 - app.routes - INFO - Serving file: /home/yair/projects/Yair_propagation/pipeline/Outputs/Archives/multiple_results.zip
2024-12-04 13:03:39,167 - app.app_main - INFO - Application is shutting down
